build.sh

```diff
@@ -5,7 +5,7 @@ mkdir -p build
 cd build
 
 # you can specify a custom install location and the python version
-cmake ../src -DCMAKE_INSTALL_PREFIX=~/open3d_install #-DPYTHON_EXECUTABLE=/usr/bin/python
+cmake ../src -DCMAKE_INSTALL_PREFIX=~/open3d_install -DPYTHON_EXECUTABLE=/usr/bin/python3.5
```

Comment:
Is it necessary to specify python version?

Reply:
good catch!
I needed that in order to generate a binary for each python version on Ubuntu.
Fixed.